coordinatorlib-soql/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/DoubleRep.scala

```diff
@@ -17,12 +17,12 @@ class DoubleRep(val base: String) extends RepUtils with SqlPKableColumnRep[SoQLT
     start + 1
   }
 
-  def dbl(v: SoQLValue) = v.asInstanceOf[SoQLDouble].value
+  def dbl(v: SoQLValue): Double = v.asInstanceOf[SoQLDouble].value
 
   def sql_in(literals: Iterable[SoQLValue]): String =
     literals.iterator.map(dbl).mkString(s"($base in (", ",", "))")
 
-  def count = "count(" + base + ")"
+  def count: String = "count(" + base + ")"
```

Comment:
Yes, but they (the reps in this dir) do not all explicitly define it. I don't know why.